modules/aggregations/src/internalClusterTest/java/org/elasticsearch/aggregations/bucket/TimeSeriesAggregationsIT.java

```diff
@@ -175,206 +148,6 @@ public void setupSuiteScopeCluster() throws Exception {
         indexRandom(true, false, docs);
     }
 
-    public void testStandAloneTimeSeriesAgg() {
```

Comment:
All these tests rely on the fact that the key (tsid) is a map of key/values pairs...which is not the case anymore.

Reply:
I don't think these tests should be removed. Maybe change the assertions? For example I do think we want to tests what the behaviour is of time_series and global aggs.

Reply:
Look I tried! This looks almost impossible...it uses something like

```
Map<Map<String, String>, Map<Long, Map<String, Double>>>
```

I ahve no idea what all these maps are mapping.

A Map with three nested maps...no idea what this is doing.
I think it is grouping by dimension values...but at the moment there is no way to know the tsid given a set of dimensions.

Reply:
Maybe the `data` can be changed to contain hashed tsids?